components/canvas.py

```diff
@@ -204,28 +208,66 @@ def mouse_to_plane(self, x, y, plane_normal, plane_offset, local_transform = Fal
             return None
         return ray_near + t * ray_dir
 
-    def orbit(self, p1x, p1y, p2x, p2y):
-        pass
-
-    def handle_rotation(self, event):
+    def zoom_test(self, factor, to = None):
```

Comment:
Is this function just for testing? If it is not, can we change the name of the function? If it is, do we need this for later?

Reply:
This is the zoom function taken from Printrun, and it's left in because I'm trying to understand its functionality. I will try to remove/refactor it next pull request.